app/javascript/app/components/intro/intro-styles.scss

```diff
@@ -38,6 +38,10 @@
   margin-bottom: 20px;
   line-height: $line-height-medium;
   color: $white;
+
```

Comment:
It actually looks nicer, but I'm not really sure if that was what they were asking for.
That's what Ola said: `On the homepage, the line break just below 'View Country Profiles' needs spacing, to be consistent with the rest.`
There's no `br` under 'View Country Profiles'.
Any way it could stay like this.

Reply:
That point I didn't really understand... because of what you said. But I used the BR in the other sections that were updated now.